### DISCUSSION TITLE
Does Willow support xtts v2 models?

### DISCUSSION THREAD

**Discussion by ther3zz** [toverainc/willow#363] [Q&A] [answered]
Hi, I was just wondering if willow supports fine tuned xtts v2 models?
Thanks!

**Comment by nikito** [comment, marked as answer]
The feature/split_arch branch of WIS supports using xtts-streaming-server, which supports voice cloning, not sure if that is what your are looking for? Note you would need to run your own WIS to really take advantage of that. 🙂

**Reply by ther3zz** [reply]
Yup, thats basically what I was looking for :)
Just wanted to use my fine tuned model with home assistant basically :)
Thanks!
